src/components/send/steps/confirm/confirm.test.js

```diff
@@ -84,4 +84,28 @@ describe('Confirm Component', () => {
       expect(wrapper.state('amount').value).to.equal(0.1);
     });
   });
+
+  describe('With account hardwareWallet account', () => {
+    beforeEach(() => {
+      props.address = '123L';
+      props.amount = 1;
+      props.accountInit = false;
+      props.account.hwInfo = { deviceId: '123123' };
+      props.pendingTransactions = ['123'];
```

Comment:
I know it was in the other test as well, but `props.xxx = yyy` is a bad practice, because it changes the object defined outside of this `describe` block, so that the blocks are not independant - this one can depend on what was set in the previous one.

You can rather use spread operator to construct the new `props` directly in the `mount`